src/main/xml-parsers.js

```diff
@@ -268,10 +276,7 @@ export function parseListObjects(xml) {
   xmlobj = xmlobj.ListBucketResult
   if (xmlobj.IsTruncated && xmlobj.IsTruncated === 'true') xmlobj.isTruncated = true
```

Comment:
@ebozduman `xmlobj.IsTruncated` is returned as boolean not as a string by the parser. so we could do
```
if (xmlobj.IsTruncated && (xmlobj.IsTruncated === true || xmlobj.IsTruncated === 'true'))
```
This is what leads to issue #881

Reply:
`xmlobj.NextContinuationToken[0]` needs to be fixed as well

Reply:
Sorry @kanagarajkm 
This part has already changed.

Reply:
@kanagarajkm 
PTAL again. I also addressed issue #881.